optuna/storages/_rdb/models.py

```diff
@@ -33,7 +34,14 @@
 
 NOT_FOUND_MSG = "Record does not exist."
 
-BaseModel: Any = declarative_base()
+convention = {
+    "ix": "ix_%(column_0_label)s",
+    "uq": "uq_%(table_name)s_%(column_0_name)s",
+    "fk": "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
+    "pk": "pk_%(table_name)s",
+}
+metadata = MetaData(naming_convention=convention)
+BaseModel: Any = declarative_base(metadata=metadata)
```

Comment:
See [here](https://alembic.sqlalchemy.org/en/latest/naming.html#integration-of-naming-conventions-into-operations-autogenerate).

Reply:
I did not specify the `ck` since it requires manually add a unique name for each constraint.

Reply:
Introducing this `metadata` may lead to unpredictable migration errors. We decide not to use this now. (I will create another discussion issue.)